Lection 3/task 3.1/ReportApp/Interfaces/IActivity/IActivityReportData.cs

```diff
@@ -0,0 +1,12 @@
+﻿using ClosedXML.Report;
+using ReportApp.Models.Activity;
+using ReportApp.Models.Entity;
+
+namespace ReportApp.Interfaces.IActivity;
+
+public interface IActivityReportData
```

Comment:
Naming issue. If it's an interface of future service - it should be named like Service

Reply:
test comment

Reply:
sooqa blyat

Reply:
report